tests/test_updater_root_rotation_integration.py

```diff
@@ -219,6 +219,80 @@ def test_root_rotation(self):
 
 
 
+  def test_BUG_ROOT_NOT_VERIFIED_WITH_CURRENT_KEYIDS_AND_THRESHOLD(self):
```

Comment:
Hold on: according to the documentation, this test should fail. Why doesn't it?

Reply:
Good catch, the doc string needs updating now that the fix is implemented.